millo.js

```diff
@@ -21,4 +21,134 @@ function listBoards() {
\ No newline at end of file
+function createBoard(boardName) {
+    for (let check in hello) {
+        if (check == boardName) {
+            return "Board already exists";
+        }
+    }
+
+    hello[boardName] = {}
+    return `Board "${boardName}" was created`;
+};
+
+function removeBoard(boardName) {
+    for (let check in hello) {
+        if (check == boardName) {
+            delete hello[boardName];
+            return `Board ${boardName} was removed`;
+        }
+    }
+    return "Board does not exist";
+};
+
+function displayBoard(boardName) {
+    let board = hello[boardName];
+    if (board === undefined) {
+        return "Board not found";
+    } else {
+        for (let content in board) {
+            console.log(`\n--------------\n${content}\n--------------\n`);
+            for (let item of board[content]){
+                console.log(`> ${item}`);
+            }
+        }
+    }
+};
+
+function createList(boardName, listName) {
+    let board = hello[boardName];
+    if (board === undefined) {
+        return "Board not found";
+    } else { 
+        let list = board[listName];
+        if (list === undefined) {
+            hello[boardName][listName] = [];
+            return `List ${listName} was added to ${boardName}`
+        } else {
+            return "List name already exists";
+        }
+    }
+}
+
+function createCard(boardName, listName, cardName) {
+    let board = hello[boardName];
+    if (board === undefined) {
+        return "Board does not exist";
+    } else {
+        let list = board[listName];
+        if (list === undefined) {
+            return "List does not exist";
+        } else {
+            hello[boardName][listName].push(cardName);
+            return `Card ${cardName} pushed to list ${listName}`;
+        }
+    }
+}
+
+function removeList(boardName, listName) {
+    let board = hello[boardName];
+    if (board === undefined) {
+        return "Board does not exist";
+    } else {
+        let list = board[listName];
+        if (list === undefined) {
+            return "List does not exist";
+        } else {
+            delete board[listName];
+            return `List ${listName} was deleted from ${boardName}`;
+        }
+    }
+}
+
+function removeCard(boardName, listName, cardIndex) {
+    let board = hello[boardName];
+    if (board === undefined) {
+        return "Board does not exist";
+    } else {
+        let list = board[listName];
+        if (list === undefined) {
+            return "List does not exist";
+        } else {
+            let cardName = hello[boardName][listName][cardIndex];
+            if (cardName === undefined) {
+                return "Card does not exist";
+            } else {
+                hello[boardName][listName].splice(cardIndex,1);
+                return `Card ${cardName} was deleted from list ${listName}`;
+            }
+        }
+    }
+}
+
+function moveCard(boardName, fromList, toList, fromCardIndex, toCardIndex) {
```

Comment:
whenever you have more than 1-2 levels of nesting, try to extract methods to allow you to reduce  it.

Here's one example of how extracting a method `getList` results in a flatter `moveList`:

```
  getList(boardName, listName) {
    return this.boards[boardName] && this.boards[boardName][listName]
  }

  moveCard(boardName, fromListName, toListName, fromCardIndex, toCardIndex) {
    if( ! this.boards[boardName]) {
      return "Board doesn't exist"
    }

    const fromList = this.getList(boardName, fromListName)
    if( ! fromList ) {
      return "From List doesn't exist"
    }
    if( fromCardIndex >= fromList.length ) {
      return "From Card doesn't exist"
    }

    const toList = this.getList(boardName, toListName)
    if( ! toList ) {
      return "To List doesn't exist"
    }

    const card = fromList[fromCardIndex]
    fromList.splice(fromCardIndex, 1)
    toList.splice(toCardIndex, 0, card)
    return 'card was moved';
  }
```